src/openrct2-ui/windows/Guest.cpp

```diff
@@ -22,6 +22,7 @@
 #include <openrct2/input.h>
 #include <openrct2/management/Marketing.h>
 #include <openrct2/peep/Staff.h>
+#include <openrct2/peep/Peep.h>
```

Comment:
This include should be above `<openrct2/peep/Staff.h>` (as we put them alphabetically).